examples/common/make-image

```diff
@@ -1,12 +1,47 @@
-#!/bin/sh
+#!/bin/bash
 
 set -eux
 
-output="$1"
+FIX_VERITY=0
+SETUP_REPART=0
+FS_FORMAT=ext4
+parsecmd() {
```

Comment:
This commandline interface thing seems a bit overkill to be honest...

It might be nicer to just be able to set a couple of extra envvars to modify the test environment, from outside:
 - an envvar to disable fs-verity complete, impacting both systemd-repart and the fix-verity script
 - an envvar to let choosing a different type for the rootfs (xfs, etc.)

Maybe we could avoid having a whole separate test scenario...